cmd/commands/root.go

```diff
@@ -31,10 +31,10 @@ var RootCmd = &cobra.Command{
 	Long: `kubeaudit is a program that makes sure all your containers are secure #patcheswelcome
 
 kubeaudit has three modes:
-1. Manifest mode: If a Kubernetes manifest file is provided using the --manifest/-f flag, kubeaudit will audit the manifest file.
+1. Manifest mode: If a Kubernetes manifest file is provided using the -f/--manifest flag, kubeaudit will audit the manifest file.
```

Comment:
I might throw in a note about it updating the manifest unless you specify an output value here!

Reply:
Actually this is just talking about the audit, not the autofix, so it doesn't modify the file. Do you think I should add a note about autofix?

Reply:
Yeah, I think that makes more sense!

Reply:
You could always throw that in a follow up PR, I don't want to block this one!